packages/suite/src/support/messages.ts

```diff
@@ -5437,11 +5437,16 @@ export default defineMessages({
     },
     TR_ADD_TOKEN_TITLE: {
         id: 'TR_ADD_TOKEN_TITLE',
-        defaultMessage: 'Add ERC20 token',
+        defaultMessage: 'Add token',
+    },
+    TR_ADD_TOKEN_DESCRIPTION: {
+        id: 'TR_ADD_TOKEN_TITLE',
+        defaultMessage:
+            "Enter a contract address of a token you wish to add to your wallet. This is typically a 42-character alphanumeric string starting with '0x'.",
```

Comment:
Please find out if we can enable it also for Solana and Cardano

Reply:
Unfortunately we can't just enable it for Solana and Cardano now since it's not working out of the box. Created an issue for it https://github.com/trezor/trezor-suite/issues/11141